examples/nested_pipelines.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+
+from neuraxle.base import Identity
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.sklearn import SKLearnWrapper
+
+
+def main():
+    # Create and fit the pipeline:
+    pipeline = Pipeline([
+        SKLearnWrapper(StandardScaler()),
+        Identity(),
+        Pipeline([
+            Identity(),  # Note: an Identity step is a step that does nothing.
+            Identity(),  # We use it here for demonstration purposes.
+            Pipeline([
+                Identity(),
+                Identity(),
+                SKLearnWrapper(PCA(n_components=2))
+            ])
+        ])
+    ])
+    X = np.random.randint(5, size=(2, 4))
```

Comment:
Safer to do this: 
`size=(100, 5)`

Because the data is random, the PCA might fail to converge, and having more data and more dimensions makes it less probable to fail. Or, the generated data could use a seed for the random.